localenv/mock-account-servicing-entity/app/lib/requesters.ts

```diff
@@ -1,169 +1,12 @@
 import { gql } from '@apollo/client'
```

Comment:
is this file necessary at all now given we have `packages/mock-account-service-lib/src/requesters.ts`?

Reply:
Yes, it has requests not in the lib and the ones in the lib are only used internally (in setupFromSeed). Commented more on this here: https://github.com/interledger/rafiki/pull/2380/files/4cd2646220d3024331afd6b933f3ee1b52dab6d2#r1521562424

If we want, we can move all the requests to the library. Could perhaps move the entire apollo client and admin client (from the integreation package) in and just expose the admin client. My initial assumption was the caller may want to form their own requests, such as different fields, or return the whole response vs. just the resource data. But none of those are necessarily blockers.

Thinking about it more, having an adminClient could make sense. It could even be useful outside the context of the tests and `mock-account-servicing-entity`. If we want to do that I would like to create a new task for it though.

Reply:
I agree with the fact that different implementations might require different fields/requests, its not difficult to add or change these requests at all anyway. Was just trying to see how much can be taken out of the MASE.

In any case, I think the MASE would benefit from a nice overhaul/update anyway, so we shouldn't spend too much time on it until we have a proper plan for where we want to take it.